apps/showcase/src/app/app.component.ts

```diff
@@ -39,6 +39,12 @@ export class AppComponent implements OnDestroy {
       links: [
         { url: '/sdk', label: 'Generator' }
       ]
+    },
+    {
+      label: 'Training',
+      links: [
+        { url: '/sdk-training', label: 'SDK Training' }
```

Comment:
To be discussed with the team but shouldn't we review that hierarchy to have something like 
```
Topic
  |_ What is it?
  |_ How to?
  |_ Training
  |_ Next steps

Reply:
Let's do it in another PR?

Reply:
yes sure 
it's just to add the topic for our next discussion